src/server/services/wfs/qgswfs.cpp

```diff
@@ -71,7 +72,15 @@ namespace QgsWfs
 
         if ( QSTR_COMPARE( req, "GetCapabilities" ) )
         {
-          writeGetCapabilities( mServerIface, project, versionString, request, response );
+          // Supports WFS 1.0.0
+          if ( QSTR_COMPARE( versionString, "1.0.0" ) )
```

Comment:
For now there's only 2 versions to manage. But something like `QgsProjectVersion` (for minor versions and so on) could be used instead of comparing string. What do you think?

Reply:
Not need to use a QgsProjectVersion, here the versionSTring has to be "1.0.0", if it's not the default version has to be returned.